### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TommyLike**

### CODE REVIEW

**Comment by TommyLike** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindspore-ai/infrastructure/pull/2#" title="Author self-approved">TommyLike</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindspore-ai%2Finfrastructure).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/mindspore-ai/infrastructure/blob/master/OWNERS)~~ [TommyLike]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by TommyLike** [comment]
@TommyLike: you cannot LGTM your own PR.

<details>

In response to [this](https://github.com/mindspore-ai/infrastructure/pull/2#issuecomment-569047838):

>/lgtm


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: README.md [hunk 1]
```diff
@@ -81,8 +81,7 @@ It's a simple repo deployment that served with nginx, now it's only for developi
 ## Official Website
 It's mindspore's official website, please visit https://wwww.mindspore.cn
 
-## CI Bot System
-It's very similar to k8s' [prow](https://github.com/kubernetes/test-infra) system, but this component is designed to handle
-gitee requests.
+## Prow System
+It's k8s' [prow](https://github.com/kubernetes/test-infra) system, but some feature are trimed.
```
Comment (leonwanghui) [inline README.md:85]: s/feature/features

Comment (hannibalhuang) [inline README.md:85]: s/k8s/kubernetes

File: README.md [hunk 2]
```diff
@@ -81,8 +81,7 @@ It's a simple repo deployment that served with nginx, now it's only for developi
 ## Official Website
 It's mindspore's official website, please visit https://wwww.mindspore.cn
 
-## CI Bot System
-It's very similar to k8s' [prow](https://github.com/kubernetes/test-infra) system, but this component is designed to handle
-gitee requests.
```
Comment (hannibalhuang) [inline README.md:86]: why there is a line change between "handle" and "gitee requests" ?

**Comment by TommyLike** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindspore-ai/infrastructure/pull/2#issuecomment-569159881" title="Approved">leonwanghui</a>*, *<a href="https://github.com/mindspore-ai/infrastructure/pull/2#" title="Author self-approved">TommyLike</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindspore-ai%2Finfrastructure).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/mindspore-ai/infrastructure/blob/master/OWNERS)~~ [TommyLike]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: README.md [hunk 3]
```diff
@@ -81,8 +81,7 @@ It's a simple repo deployment that served with nginx, now it's only for developi
 ## Official Website
 It's mindspore's official website, please visit https://wwww.mindspore.cn
 
-## CI Bot System
-It's very similar to k8s' [prow](https://github.com/kubernetes/test-infra) system, but this component is designed to handle
```
Comment (hannibalhuang) [inline README.md:85]: s/k8s/kubernetes

**Pull request closed without merging**